Fix typos

GitHubPang opened the pull request:
* IoC is **I**nversion **o**f **C**ontrol, so for Dry**I**oc, it's a capital **i** and not a small **L**.
* Also fixed one other typo.

GitHubPang committed:
Fix typo (DryIoc instead of Dryloc)

GitHubPang committed:
Fix typo in modules.md

dansiegel reviewed (changes requested):
Please remove any references to Ninject as we no longer support the Ninject Container. Great catch btw!

GitHubPang committed:
Removed references to Ninject

https://github.com/PrismLibrary/Prism-Documentation/pull/89#pullrequestreview-545674955

GitHubPang committed:
Merge remote-tracking branch 'upstream/master'

dansiegel reviewed (approved):
Thanks for the fixes!
